controllers/PasswordReset.js

```diff
@@ -0,0 +1,83 @@
+import shortId from 'shortid';
+import { User } from '../models';
+import sendMail from '../helpers/emails';
+
+const { HOST_URL } = process.env;
+
+/**
+ * Password reset class
+ */
+class PasswordReset {
+  /**
+   * @description Check if user exists and is verified
+   * @param {string} email user email
+   * @returns {Boolean} true or false
+   */
+  static async userExists(email) {
```

Comment:
Consider extracting this to a helper function, so it can be available for others to use.